cfme/test_requirements.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+alert = pytest.mark.requirement("alert")
```

Comment:
Can we put this in the markers module, seems like as they are markers it would make more sense there?